README.md

```diff
@@ -4,10 +4,12 @@ This repo has a "Hello World" Go application that uses the [pgx driver](https://
 
 To run the code:
 
-1. Start a [local, insecure CockroachDB cluster](https://www.cockroachlabs.com/docs/stable/start-a-local-cluster.html).
+1. Start a [demo CockroachDB cluster](https://www.cockroachlabs.com/docs/stable/cockroach-demo.html).
```

Comment:
Minor thing here, but the linked page doesn't document `cockroach demo --empty`, so starting the cluster will populate `movr`.

Reply:
Added a command string.